### PR TITLE
Packages updated

### PR DESCRIPTION
**Pull request by babenek**
## Description

Please include a summary of the change and which is fixed.

- Update packages for Python3.8 on 2024.01.03
- Fix benchmark scores

## How has this been tested?

Please describe the tests that you ran to verify your changes.

- [x] UnitTest
- [x] Benchmark
- [x] TODO: rollback custom BM references after https://github.com/Samsung/CredData/pull/64

### CODE REVIEW

File: .github/workflows/benchmark.yml [hunk 1]
```diff
@@ -22,7 +22,8 @@ jobs:
     - name: Checkout CredData
       uses: actions/checkout@v3
       with:
-        repository: Samsung/CredData
+        ref: rndlineseed
+        repository: babenek/CredData
```
Comment (babenek) [inline .github/workflows/benchmark.yml:26]:
```suggestion
        repository: Samsung/CredData
```

File: .github/workflows/benchmark.yml [hunk 2]
```diff
@@ -62,7 +63,8 @@ jobs:
...
```
Comment (babenek) [inline .github/workflows/benchmark.yml:67]:
```suggestion
        repository: Samsung/CredData
```

**Review by babenek — COMMENTED** [review]
TODO: rollback the custom reference after https://github.com/Samsung/CredData/pull/64

File: cicd/benchmark.txt [hunk 3]
```diff
@@ -4,22 +4,22 @@ Category                      Positives    Negatives    Template
...
-                            4263   894  19373417   823  4.614e-05  0.161817   0.999911  0.826643  0.838183  0.832373
+Authentication Key & Token    58     4        28    19  0.125      0.246753   0.788991  0.935484  0.753247  0.834532
+Generic Secret               970     3       215    86  0.0137615  0.0814394  0.930141  0.996917  0.918561  0.956136
+Generic Token                285     7       596    48  0.0116086  0.144144   0.941239  0.976027  0.855856  0.912
+Other                        591   745     62982   251  0.0116905  0.2981     0.984575  0.442365  0.7019    0.5427
+Password                     986   126      4127   413  0.0296262  0.295211   0.904636  0.886691  0.704789  0.785344
+Predefined Pattern           316     2        40    14  0.0476191  0.0424242  0.956989  0.993711  0.957576  0.975309
+Private Key                 1009     3        31     0  0.0882353             0.997124  0.997036  1         0.998516
+Seed, Salt, Nonce             38     2         6     3  0.25       0.0731707  0.897959  0.95      0.926829  0.938272
+                            4253   892  19373418   834  4.604e-05  0.163947   0.999911  0.826628  0.836053  0.831314
```
Comment (babenek) [inline cicd/benchmark.txt:25]:
```suggestion
                            4253   892  19373418   834  4.604e-05  0.163947   0.999911  0.826628  0.836053  0.831314
                            
```

File: cicd/benchmark.txt [hunk 4]
```diff
@@ -1,25 +1,24 @@
...
+Authentication Key & Token           84            1          31
+Generic Secret                     1055           17         203
+Generic Token                       339           45         560
+Other                               905        62995         631
+Password                           1402          110        4143
+Predefined Pattern                  347            2          39
+Private Key                        1009            2          31
+Seed, Salt, Nonce                    44            4           4
+TOTAL:                             5185        63176        5642
+credsweeper result_cnt : 5145, lost_cnt : 0, true_cnt : 4355, false_cnt : 790
```
Comment (babenek) [inline cicd/benchmark.txt:13]:
```suggestion
Detected Credentials: 5913
credsweeper result_cnt : 5145, lost_cnt : 0, true_cnt : 4355, false_cnt : 790
```

**Review by kmnls — APPROVED** [review]
Agree

**Review by iuriimet — APPROVED** [review]

**Pull request merged**